# Generated by Django 4.1.5 on 2023-02-07 14:39

import datetime
from django.conf import settings
import django.core.validators
from django.db import migrations, models
import events.models


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('events', '0002_alter_participation_event_alter_participation_person'),
    ]

    operations = [
        migrations.AlterField(
            model_name='event',
            name='dateEvent',
            field=models.DateField(validators=[events.models.isDateIsValid]),
        ),
        migrations.AlterField(
            model_name='event',
            name='nbrParticipants',
            field=models.ImageField(default=0, upload_to='', validators=[django.core.validators.MinValueValidator(limit_value=0, message='la valeur doit etre positif ')]),
        ),
        migrations.AlterUniqueTogether(
            name='participation',
            unique_together={('person', 'event')},
        ),
        migrations.AddConstraint(
            model_name='event',
            constraint=models.CheckConstraint(check=models.Q(('dateEvent__gte', datetime.datetime(2023, 2, 7, 15, 39, 16, 89374))), name='la date est  invalide'),
        ),
    ]
